cmd/hstspreload/main.go

```diff
@@ -116,6 +116,18 @@ includeSubDomains: %s%t%s
 				underline, domain, resetFormat,
 				bold, state.Mode, resetFormat,
 				bold, state.IncludeSubDomains, resetFormat)
+		} else if status == preloadlist.AncestorEntryFound {
+			fmt.Printf(`%s%s%s is preloaded by virtue of its ancestor domain:
```

Comment:
I think this is a good description, although I think we can make it even simpler. How about "An ancestor of example.com is preloaded"?